antlr-kotlin-runtime/build.gradle.kts

```diff
@@ -1,20 +1,20 @@
-import com.strumenta.kotlinmultiplatform.gradle.ext.*
-import org.jetbrains.kotlin.gradle.dsl.ExplicitApiMode
-import org.jetbrains.dokka.gradle.DokkaTask
+@file:Suppress("UnstableApiUsage")
+
+import com.strumenta.kotlinmultiplatform.gradle.ext.setupPom
+import com.strumenta.kotlinmultiplatform.gradle.ext.targetsNative
 import com.vanniktech.maven.publish.SonatypeHost
+import org.jetbrains.dokka.Platform
+import org.jetbrains.dokka.gradle.DokkaTask
+import org.jetbrains.kotlin.gradle.dsl.ExplicitApiMode
 
 plugins {
   id("strumenta.multiplatform")
   id("org.jetbrains.dokka")
-  id("com.vanniktech.maven.publish")
 }
 
 strumentaMultiplatform {
   applyJvm()
-
-  if (targetsJS()) {
-    applyJs()
-  }
+  applyJs()
```

Comment:
Why are we removing this?

Reply:
It was always true `anyway`, and JS should be always applied and enabled to make sure it builds and tests fine.

Reply:
ok then